ReadMe.md

```diff
@@ -1,47 +1,74 @@
+<p align="center">
+<img src="docs/images/wct.png" alt="Windows Community Toolkit Gallery"/>
```

Comment:
Looks like this image isn't rendering. This might fix it.
```suggestion
<img src="docs/images/WCT.png" alt="Windows Community Toolkit Gallery"/>
```

Reply:
I don't see an issue with its display? I know the rich diff had an issue in the past with first render, but I see it fine. It's marked as lowercase in the PR?